### ISSUE TITLE
[bug] Pylint error: Module 'random' has no 'random' member; maybe 'Random'

### ISSUE THREAD

**Issue by inlann** [microsoft/vscode-pylint#211]
Hello:

My Pylint extension throw a member error for the module 'random':

```python
'''
Test file
'''

import random

value = random.random()
print(value)
```

<img width="714" alt="image" src="https://user-images.githubusercontent.com/33852855/198217012-fa9d82b4-df6c-4e99-b821-0c4752dbe8fa.png">

My random module works fine and my Pylint does not throw any error for this script:

```bash
mac@Titans-iMac vcmb_transform % python3
Python 3.9.6 (default, Aug  5 2022, 15:21:02) 
[Clang 14.0.0 (clang-1400.0.29.102)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import random
>>> random.random()
0.38445701937620147
>>> exit()
mac@Titans-iMac vcmb_transform % python3 -m pylint  test.py

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

Pylint extension version: v2022.4.0
OS: Darwin

**Comment by karthiknadig** [comment]
@inlann Can you tell me if your project has a file called `random.py`? It would be good to diagnose if you can share your project or share your project structure. Also share the logs from Output > Pylint and Output > Python.

**Comment by karthiknadig** [comment]
This type of issue commonly occurs if you have another file by the name `random.py` in project.

**Comment by inlann** [comment]
@karthiknadig So thanks!! I remove my `random.py` and it works!!
